Resources/Prototypes/Entities/Objects/Fun/darts.yml

```
@@ -85,6 +85,9 @@
       - !type:DoActsBehavior
         acts: [ "Destruction" ]
   - type: BalloonPopper
+  - type: Tag  # Goobstation
+    tags:
+    - SyringeGunAmmo
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)